src/bot/config.ts

```diff
@@ -56,8 +56,8 @@ export const validateConfig = (config: InternalContributionForksConfig) => {
   try {
     internalContributionForksConfig.parse(config)
   } catch (e) {
-    configLogger.error('Invalid config found!')
-    configLogger.error(e)
+    configLogger.error(new Error('Invalid config found!'))
```

Comment:
Since we are building the config out of env variables, can we be more specific here about what was wrong with the config? Maybe ensure that `e` is passed through to the logger if there is parse failure info there? 

Would like to go from a `error: Invalid config found!` entry to something more like `error: Invalid config found! Unable to parse "config.publicOrg"` in order to make the error message more specific and directly actionable.